packages/react-components/react-shared-contexts/src/AnnounceContext/AnnounceContext.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export type AnnounceOptions = {
+  alert?: boolean;
+  priority?: number;
+  batchId?: string;
+};
+
+/**
+ * @internal
+ */
+export type AnnounceContextValue = (message: string, options: AnnounceOptions) => void;
```

Comment:
```suggestion
export type AnnounceContextValue<T = AnnounceOptions> = (message: string, options?: T) => void;
```
I think it'd be good to make it easy to change the options type

Reply:
also the context value should be an object that has the fn as a prop on the object, probably just called `announce`?